### ISSUE TITLE
core-js build error new promise capability requires a-function instead of a-callable

### ISSUE THREAD

**Issue by VishalGulati** [zloirock/core-js#1282]
I have core-js 3.21.1 in my React application and the build sometimes fails due to core-js requiring a-function inside new-promise-capability instead of a-callable.

Per the git code for core-js too, it should import from a-callable instead of a-function inside internals. Since there is no a-function inside internals, my build fails.

In my babel config json I have babel preset env fixed to corejs 3. What could be causing 2 different corejs code for new-promise-capability when the version of core-js is fixed to 3.21.1.

When build works, new-promise-capability reads:
`var aCallable = require('../internals/a-callable');`

When it fails, it reads:
`var aFunction = require('../internals/a-function');`

**Comment by VishalGulati** [comment]
@zloirock - Appreciate your insights here, but my code uses 3.21.1 only and in node_modules/core-js/package.json too I can see 3.21.1 being installed. Why is then the internal script "new-promise-capability.js" not using the correct code? Please share if you have any input on this.

**Comment by VishalGulati** [comment]
Yeah, I mentioned in my question too that the current git code or 3.21.1 too doesn't import from a-function. But since my node_modules/core-js/package.json confirms 3.21.1 is installed then how come this obsolete code present in internal scripts?

**Comment by zloirock** [comment]
That means that you have `core-js` not only in `node_modules/core-js`.

**Comment by VishalGulati** [comment]
Unfortunately, the is no other reference of core-js apart from package.json where its 3.21.1.

**Comment by zloirock** [comment]
There are no other options. Otherwise, this module has nowhere to take.

**Comment by VishalGulati** [comment]
Its happening on just 1 server and not all, had it been the case, it would have happened at all servers.
